bus_presence_override/static/src/js/systray.js

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2017 LasLabs Inc.
+   License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+odoo.define('bus_presence_override.systray', function (require) {
```

Comment:
IMO you should use something more descriptive than `systray`. This is only for the editing of an IM status, and is not a system tray. `bus_presence_selection` or something